reference/customers.v2.yml

```diff
@@ -1375,23 +1393,31 @@ components:
                   - price
               amount:
                 type: string
-                description: A float that specifies the value applied to the price modified
+                description: A float that specifies the value applied to the price modified.
                 example: '"5.0000" (Float, Float as String, Integer)'
               price_list_id:
                 type: integer
-                description: 'If a customer group is assigned to a price list,`method` and `amount` are not shown. `type` and `price_list_id` are returned.'
+                description: If a customer group is assigned to a price list,`method` and `amount` are not shown. `type` and `price_list_id` are returned.
                 example: 3
+        date_created:
+          type: string
+          description: Date on which the customer created the group.
+          example: 2023-07-17 06:30:41
+        date_modified:
+          type: string
+          description: Date on which the customer last modified the group.
+          example: 2023-07-25 01:15:19
         is_group_for_guests:
           type: boolean
           description: If the groups is for guests. There can only be one customer group for guests at a time.
-      description: 'When creating a customer group category discount using the API it defaults to "products in this category and its subcategories". In the [Control Panel](https://support.bigcommerce.com/s/article/Customer-Groups#pricing) this can be changed to either "products in this category only" or "products in this category and its subcategories". There are currently no settings to change this behavior via API.'
+      description: When creating a customer group category discount using the API it defaults to "products in this category and its subcategories". In the [Control Panel](https://support.bigcommerce.com/s/article/Customer-Groups#pricing) this can be changed to either "products in this category only" or "products in this category and its subcategories". There are currently no settings to change this behavior with the API.
```

Comment:
same

Reply:
```suggestion
      description: When creating a customer group category discount using the API it defaults to "products in this category and its subcategories". In the [store control panel](https://support.bigcommerce.com/s/article/Customer-Groups#pricing), this can be changed to either "products in this category only" or "products in this category and its subcategories". There are currently no settings to change this behavior with the API.
```